### DISCUSSION TITLE
Como instalar Boss no linux?

### DISCUSSION THREAD

**Discussion by FernandoMassa** [HashLoad/boss#161] [Q&A]
Nas instruções de instalação do Boss só tem um exemplo banal de Windows, mas como faço pra instalar ele no Linux? baixei a versão do linux e no readme.md tem o mesmo exemplo de instalação pra Windows.
Alguém poderia me ajudar? não tenho muita experiência com Linux neste tipo de coisas. 
Agradeço a paciência e desculpem minha ignorância.

**Comment by juliomar** [comment]
Creio que só colocar em uma pasta e dar permissões a ele.
